ui/chapter/Chapter.tsx

```diff
@@ -140,6 +142,10 @@ export default function Chapter({ children, metadata, lang }) {
             {t(chapter.metadata.title)}
           </h3>
 
+          {chapterWeAreRendering?.hasDifficulty && (
```

Comment:
We should probably also make sure this doesn't render until that chapter has been unlocked